x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/profile/ProfileService.java

```diff
@@ -191,7 +192,7 @@ public void searchProfile(SearchProfilesRequest request, ActionListener<SearchPr
                         "user_profile.user.full_name._2gram",
                         "user_profile.user.full_name._3gram",
                         "user_profile.user.email"
-                    ).type(MultiMatchQueryBuilder.Type.BOOL_PREFIX)
+                    ).type(MultiMatchQueryBuilder.Type.BOOL_PREFIX).fuzziness(Fuzziness.AUTO)
```

Comment:
The addition of `fuzziness` is not really about renaming. But it is too trivial to be on its own PR.

Reply:
On a second thought, I think we should leave it in a future PR after discussing whether we do want it. So I dropped it from this PR.